test/test.md

```diff
@@ -8,5 +8,6 @@
 - [[sub.md]] - exists in sub directory
 - [[extension test mirror]]
 - [[extension-test-mirror.md]]
+- [](test) - hyperlink (should show up in the backlinks panel)
```

Comment:
We should probably note here that we allow other VS Code extensions / defaults handle the hyperlink completion, just to be clear with people that this is out of our scope.